metrics/prometheus/prometheus.go

```diff
@@ -0,0 +1,272 @@
+package metrics
+
+import (
+	"github.com/prebid/prebid-cache/config"
+	"github.com/prometheus/client_golang/prometheus"
+	"time"
+)
+
+/**************************************************
+ * Constants and global variables
+ **************************************************/
+var TenSeconds time.Duration = time.Second * 10
```

Comment:
Since the structure of the metrics engine requires both Influx and prometheus to use the same constant strings, wouldn't this best be extracted to a singular `metrics_constants`, `metrics_util`, or some other shared package so you can be sure they stay in sync? This is what PBS does, it has two global metrics packages, the main one that will import all the metrics engines, and a second one for shared resources that all the metrics engines can import.

And again, variables instead of constants?

Reply:
No need to export them to a separate file becauyse prometheus and influx are now using different constants. Refactored and changed them to constants.

Reply:
No, we should be good here now.